apps/user/serializers/user.go

```diff
@@ -34,3 +34,9 @@ type TeamsSerializer struct {
 type MembersSerializer struct {
 	Members []User
 }
+
+// UserLogin ...
+type UserLogin struct {
+	Email    string `json:"Email"`
```

Comment:
Shouldn't the JSON fields be lower-case?

Reply:
done.